Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/EntityPropertyEditor.hxx

```diff
@@ -245,24 +247,29 @@ namespace AzToolsFramework
         void OnEntityStartStatusChanged(const AZ::EntityId& entityId) override;
         //////////////////////////////////////////////////////////////////////////
 
-        // EditorComponentModeNotificationBus
-        void ActiveComponentModeChanged(const AZ::Uuid& componentType) override;
-
         // ViewportEditorModeNotificationsBus overrides ...
         void OnEditorModeActivated(
             const AzToolsFramework::ViewportEditorModesInterface& editorModeState, AzToolsFramework::ViewportEditorMode mode) override;
         void OnEditorModeDeactivated(
             const AzToolsFramework::ViewportEditorModesInterface& editorModeState, AzToolsFramework::ViewportEditorMode mode) override;
 
-        // EntityPropertEditorRequestBus
+        // EditorComponentModeNotificationBus overrides ...
+        void ActiveComponentModeChanged(const AZ::Uuid& componentType) override;
+
+        // Save the offset of the scroll bar before entering Component Mode to return to after leaving Component Mode
+        AZStd::optional<int> m_verticalScrollOffset;
+        // Save the offset of scroll bar when Component Mode has been entered
+        AZStd::optional<int> m_componentModeVerticalScrollOffset;
```

Comment:
update: Should these now be delete now? I don't see them referenced anywhere else

Reply:
Thank you for spotting this!